libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -914,6 +914,7 @@
       "download": "Download",
       "drawer": {
         "edit": "Rediger rettighed",
```

Comment:
suggestion: the edit button in user drawer also shows incorrect text value (in Danish). Can we fix that too :)

Reply:
Sure :)